### ISSUE TITLE
Many missing examples, still welcoming contributions?

### ISSUE THREAD

**Issue by opfuchs** [aimacode/aima-csharp#10]
I know this was in the closed issues last year, but is the project still alive? I thought doing AIMA code in C# was a pet idea of mine until I stumbled upon this tonight while browsing the AIMA repos.

Are you all still welcoming contributions? I'm putting the finishing touches on the table vacuum agent and the vacuum environment tonight and tomorrow, and then will be focusing on the searches.

**Comment by ShobhitLamba** [comment]
Yes, the project is always open to more contributions and more contributors. The repo does not have many contributors so it is mostly dormant, but make a pull request and we will be more than happy to review and accommodate it.
